examples/plot_sleep_staging_usleep.py

```diff
@@ -0,0 +1,360 @@
+"""
+Sleep staging on the Sleep Physionet dataset using U-Sleep network
+==================================================================
+
+.. note::
+    Please take a look at the simpler sleep staging example
+    `here <https://braindecode.org/auto_examples/plot_sleep_staging.html>`_
+    before going through this example. The current example uses a more complex
+    architecture and a sequence-to-sequence (seq2seq) approach.
+
+This tutorial shows how to train and test a sleep staging neural network with
+Braindecode. We adapt the U-Sleep approach of [1]_ to learn on sequences of EEG
+windows using the openly accessible Sleep Physionet dataset [2]_ [3]_.
+
+References
+----------
+.. [1] Perslev M, Darkner S, Kempfner L, Nikolic M, Jennum PJ, Igel C.
+       U-Sleep: resilient high-frequency sleep staging. npj Digit. Med. 4, 72 (2021).
+       https://github.com/perslev/U-Time/blob/master/utime/models/usleep.py
+
+.. [2] B Kemp, AH Zwinderman, B Tuk, HAC Kamphuisen, JJL Oberyé. Analysis of
+       a sleep-dependent neuronal feedback loop: the slow-wave
+       microcontinuity of the EEG. IEEE-BME 47(9):1185-1194 (2000).
+
+.. [3] Goldberger AL, Amaral LAN, Glass L, Hausdorff JM, Ivanov PCh,
+       Mark RG, Mietus JE, Moody GB, Peng C-K, Stanley HE. (2000)
+       PhysioBank, PhysioToolkit, and PhysioNet: Components of a New
+       Research Resource for Complex Physiologic Signals.
+       Circulation 101(23):e215-e220
+"""
+# Authors: Theo Gnassounou <theo.gnassounou@inria.fr>
+#          Omar Chehab <l-emir-omar.chehab@inria.fr>
+#
+# License: BSD (3-clause)
+
+
+######################################################################
+# Loading and preprocessing the dataset
+# -------------------------------------
+#
+# Loading
+# ~~~~~~~
+#
+# First, we load the data using the
+# :class:`braindecode.datasets.sleep_physionet.SleepPhysionet` class. We load
+# two recordings from two different individuals: we will use the first one to
+# train our network and the second one to evaluate performance (as in the `MNE`_
+# sleep staging example).
+#
+# .. _MNE: https://mne.tools/stable/auto_tutorials/sample-datasets/plot_sleep.html
+#
+
+from braindecode.datasets.sleep_physionet import SleepPhysionet
+
+dataset = SleepPhysionet(
+    subject_ids=[0, 1], recording_ids=[2], crop_wake_mins=30)
+
+
+######################################################################
+# Preprocessing
+# ~~~~~~~~~~~~~
+#
+# Next, we preprocess the raw data. We scale each channel recording-wise to
+# have zero median and unit interquartile range. We don't upsample to 128 Hz as
+# done in [1]_ so that we keep the example as light as possible. No filtering
+# is described in [1]_.
+
+from braindecode.preprocessing import preprocess, Preprocessor, zscore
+
+# XXX To be changed with robust_scale
+preprocessors = [Preprocessor(zscore, channel_wise=True)]
```

Comment:
so here could be possible to just use scale from sklearn as per https://github.com/braindecode/braindecode/pull/291#issuecomment-872072222 ? or would that be different?

Reply:
but could also handle this in separate PR I guess? Then I would merge? Unless you want to change this here already

Reply:
As the comment above suggests, it should be `sklearn.preprocessing.robust_scale`. But yes, it should be possible to use this here instead.

Reply:
`sklearn.preprocessing.robust_scale` or `sklearn.preprocessing.scale`?